Source/DataModels/Components/ComponentTransform.h

```diff
@@ -41,11 +41,13 @@ class ComponentTransform : public Component, public Drawable
 	const AABB& GetEncapsuledAABB();
 	const OBB& GetObjectOBB();
 	bool IsDrawBoundingBoxes() const;
+	bool IsUniformScale() const;
 
 	void SetPosition(const float3& position);
 	void SetRotation(const float3& rotation);
 	void SetRotation(const float4x4& rotation);
 	void SetScale(const float3& scale);
+	void SetUniformScale(const float3& scale, const char modifiedScaleAxis);
```

Comment:
```suggestion
	void SetUniformScale(const float3& scale, char modifiedScaleAxis);
```
Not much of a point in having a `const` primitive type.